vendor/omnipay/eway/src/Message/RapidResponse.php

```diff
@@ -52,4 +52,12 @@ public function getCardReference()
 
         return null;
     }
+
+    /**
+     * Get InvoiceNumber - merchant reference for a transaction
+     */
+    public function getInvoiceNumber()
```

Comment:
no no no - we need to convince them to use the standardised method name for this getTransactionId